lasagne/objectives.py

```diff
@@ -341,45 +342,50 @@ def multiclass_hinge_loss(predictions, targets, delta=1):
 def huber_loss(predictions, targets, delta=1):
     """ Computes the huber loss between predictions and targets.
 
-    .. math:: L_\delta (diff) = \frac{diff^2}{2} & \text{if  } |diff| \le \
-                                \delta
-    .. math:: L_\delta (diff) & = & \delta (|diff| - \frac{\delta}{2} ),\
+    .. math:: 
+        L_\delta (diff) = \frac{diff^2}{2} & \text{if  } |diff| \le \ delta \\
+
+        L_\delta (diff) & = & \delta (|diff| - \frac{\delta}{2} ),\ \\
                                      &\text{else}
 
     Parameters
     ----------
-    predictions : Theano 2D tensor
-        Predictions in (0, 1), such as softmax output of a neural network,
-        with data points in rows and class probabilities in columns.
+    predictions : Theano 2D tensor or 1D tensor
+        Predictions output of a neural network
+
     targets : Theano 2D tensor or 1D tensor
-        Either a vector of int giving the correct class index per data point
-        or a 2D tensor of one-hot encoding of the correct class in the same
-        layout as predictions (non-binary targets in [0, 1] do not work!)
+        Ground truth to which the prediction is to be compared
+        with.
+
     delta : scalar, default 1
         This delta value is defaulted to 1, for SmoothL1Loss
-        described in Fast-RCNN paper[1].
+        described in Fast-RCNN paper[1]_.
 
     Returns
     -------
     Theano 1D tensor
-        An expression for the item-wise huber loss.
+        An expression for the element-wise huber loss.
 
     Notes
     -----
-    This is an alternative to the Least Squared loss for
+    This is an alternative to the squared error for
     regression problems.
 
     References
     ----------
     .. [1] Ross Girshick et al (2015):
            Fast RCNN
            https://arxiv.org/pdf/1504.08083.pdf
+
+    .. [2] Huber, Peter et al (1964)
```

Comment:
Ah, didn't see the comment about your browser problem. Do you have another browser you can try? (Using another computer seems overkill!) Otherwise I can also check out your PR and test the documentation here, but of course it's easier if you can debug and fix things directly!